elyra/pipeline/processor_kfp.py

```diff
@@ -51,7 +51,7 @@ def process(self, pipeline):
         cos_bucket = runtime_configuration.metadata['cos_bucket']
 
         user_namespace = None
-        if runtime_configuration.metadata['user_namespace']:
+        if runtime_configuration.metadata.get('user_namespace'):
             user_namespace = runtime_configuration.metadata['user_namespace']
```

Comment:
Damn - I'm sorry.  I spaced that using `.get('non-existent-key')` will return `None`!  So all you need (as Luciano pointed out long ago :blush: ) is the assignment statement:
```suggestion
            user_namespace = runtime_configuration.metadata.get('user_namespace')
```
with out the `if` statement.  I apologize.

Reply:
removed and updated